src/Util/setHighlight.ts

```diff
@@ -10,12 +10,12 @@ const setHighlight = (
   clearHighlightRef: number,
   setClearHighlightFn: Function,
   updateClearHandler: Function,
-  uniqueId: number
+  uniqueId: string
 ) => {
 
   const element = child.ref.current;
 
-  if (!isInitial && element && parseInt(element.getAttribute(defaults.HIGHLIGHT_UNIQUE_ID)) === uniqueId) {
+  if (!isInitial && element && element.getAttribute(defaults.HIGHLIGHT_UNIQUE_ID) === uniqueId) {
```

Comment:
I'v updated the version of typescript to `3.7.2` which has the optional chaining, so this like could be as follow
```js
if (!isInitial && element?.getAttribute(defaults.HIGHLIGHT_UNIQUE_ID) === uniqueId) {
```